llvm/lib/Transforms/IPO/MemProfContextDisambiguation.cpp

```diff
@@ -3472,15 +3472,19 @@ bool MemProfContextDisambiguation::applyImport(Module &M) {
             assert(StackMDNode);
             SmallVector<unsigned> StackIdsFromMetadata;
```

Comment:
I think this is dead and can be removed too.

Reply:
Done.

Reply:
My comment was about the StackIdsFromMetadata declaration, not the assert.

Reply:
Update.